src/core/a-assets.js

```diff
@@ -234,7 +234,7 @@ function extractDomain (url) {
   var domain = url.indexOf('://') > -1 ? url.split('/')[2] : url.split('/')[0];
 
   // Find and remove port number.
-  return domain.split(':')[0];
+  return domain.substring(0, domain.indexOf(':'));
```

Comment:
What about the url.split just above?

Reply:
Seems a bit unclear about doing it with string splits and didn't want to risk breaking it :)